### DISCUSSION TITLE
Help speakers stick to 20 minutes and format of lnug

### DISCUSSION THREAD

**Discussion by Ankcorn** [lnug/feedback#116] [General]
Thanks, @GiacomoSorbi for really respectfully raising this on meetup. I have copied this over to a GitHub issue as it is a better forum to have a productive discussion around how we can change things.

The comment was

> Ok, I will try to be nice and let me make clear that I have full respect for all the people committing for free to make such events possible:
 I confirm my talk will be 20 mins long 🤞🏾
This intention was definitely not confirmed and while the talk somehow derailed to a bit more than a hour, I believe it would have been better to make clear otherwise. I will never be the one complaining if a talk goes 5 mins shorter or 10 minutes longer, but going for 3X time seems a bit... Unprepared? Similarly some more rehearsing in the live coding part might have helped, but ok: maybe it was announced differently and I missed it coming a few mins late - so my bad in case.
At the end of it, also the endless break was not the best - I wanted to see Artsiom's lighting talk, but as it ended to be too late (I had a busy day incoming and sleep deprivation is not my best buddy), I had to leave with a few, similarly opinionated friends.
I would recommend enforcing the planned schedule in a stricter way for future events, or at least taking a small break, warning that things are going longer than expected, and then moving on with the ones planning to stay - 1 hour straigt long talk is not what everybody might be up to, not after a full day of work.
Thanks for reading :)
To answer the questions in the post
It was announced as a regular talk, we sometimes do have talks longer if prearranged and only with a really good reason.

On LNUG format and the break

The plan is normally
7 pm start
7.15 pm first talk starts
7.35 pm break till 8
8 pm community announcements
8.15 pm start next talk

Obviously, that went out the window tonight. If people would prefer a different format to that tell us here.

Any suggestions to help us with timekeeping during the talks?

Somebody at the pub suggested giving all speakers a 10-minute warning, 5 minutes, 2 minutes, and 0 minutes warning.
If I was doing a talk I'd actually find that super helpful, do people agree? Any alternative ideas?

Thanks again for giving us feedback 💚

**Comment by robtweed** [comment]
Unfortunately I'm already booked to give a talk on the 4th March, so I can't make the workshop on how to create a talk proposal. However, I'd be happy to provide guidance on how to prepare an effective presentation at some future workshop if that would be helpful to the community?

**Comment by Ankcorn** [comment]
@robtweed I'm sorry we have a clash!

@admataz is doing a lightning talk on talking at meetups at the start but if you have some words of wisdom to share then that would totally be appreciated! 🤘🥳

**Comment by GiacomoSorbi** [comment]
I think having a handy timer (physical or digital - up to you) in the talker sight might be enough; politely soliciting the talker to wrap things up in a few mins might also be a good option.

**Comment by robtweed** [comment]
1) My "golden rule" for any presentation is to think "so what?".  ie, what I'm going to talk about might be the coolest stuff on earth, but if everyone goes away thinking "well that sure looked cool, but so what?"  (in other words, how or why was that relevant to the audience and their problems?), then your presentation will be largely wasted on everyone.  The key is "know thy audience" and make your pitch relevant to the problems they're dealing with or solutions they might be looking for

**Comment by robtweed** [comment]
2) 20 minutes is almost *no time at all* once you're up there.  My rule of thumb for slides with text on them is 2 minutes per slide minimum - so if you're not even demo-ing anything any more than 8 - 10 slides and you'll almost certainly over-run.  A 20 minute talk is only going to allow you to cover a very small amount of information, so you must keep VERY focused on the info you're trying to impart and don't be tempted to veer off it by "busking it" on the day.  The audience may be technical, but the stuff you're talking about may be brand new to many.  Don't go into the technical weeds in the talk - concentrate on the "so what?" of your topic.  For sure, provide one or two juicy glimpses of how cool the stuff you're talking about is to hook them....but provide some pointers at the end of your presentation (or in post presentation forums) where people who you've hooked in your talk can go to find out more info.  So don't expect in 20 mins that you'll be able to explain everything there is to know about what your presentation is on!  You're there to tell a story and bring something cool to the attention of people in the audience who might not know much, if anything, about it, but who you think could benefit by knowing about it

**Comment by Ankcorn** [comment]
> I think having a handy timer (physical or digital - up to you) in the talker sight might be enough; politely soliciting the talker to wrap things up in a few mins might also be a good option.

A physical timer would totally work well good idea 👍

**Comment by Ankcorn** [comment]
>     "so what?".

Nice I love it! Thanks @robtweed

**Comment by robtweed** [comment]
3) Rehearse, rehearse, rehearse.  Personally I do this in my head rather than out loud.  I'll go through the slides and demo over and over, tuning it, focusing it.  By the time you give your presentation, you should feel like this is the nth time your've done it.  DON'T go off script on the day - you'll sink without trace.  Remember the audience have taken time out of their personal lives to come and listen to you - you owe it to them to make them go away thinking "Wow I'm really glad I went tonight - that was really helpful"  rather than "yeah that was kind of interesting but they lost me after the first 3 minutes and it didn't seem that relevant to me"

**Comment by robtweed** [comment]
4) If you're doing a demo, you're going to a) need to allow at least 10 minutes which, again, is no time at all; so b) you're going to have to be VERY focused about what you attempt to demo; and c) make sure you rehearse the demo several times over and ensure it fits into the time; d) ideally have most of the demo pre-written so you're not making mistakes and end up just showing it crashing in front of the audience.  Assume that even the best-rehearsed demo will go wrong, so have a contingency plan if it does - ie resort to a set of screen-shots or a "one I prepared earlier" video

**Comment by robtweed** [comment]
A presentation is like an ice-berg.  That 20 minutes the audience sees hides many hours of work that it took to make it effective.  A good speaker will make it seem very easy on the day.  But actually it never is.  If it looked easy, that's hard work and hard-earned experience at play!
 Experience definitely helps - the more presentations you do, the better you can become...but only if you review how your last one(s) went down.  Try to gauge the reaction and learn from any mistakes and ensure you avoid them in the next one.

**Comment by robtweed** [comment]
All that being said, don't let it put you off giving a presentation.  Like learning to drive a car, we all gave a first presentation at some time and it was one of the scariest things we ever did!  But hopefully some of my thoughts/experience above will help you avoid a car crash on your first outing! :-)

**Comment by robtweed** [comment]
On the subject of demos / live coding - once again think "so what?"  Why are you demo-ing / coding what you're doing.  What are you trying to get the audience to take away from that demo/coding?  If it doesn't add to the message, don't feel you have to demo at all - instead is there somewhere you can send the audience too afterwards where they can quickly and easily try it for themselves?  Or point them at a video you've previously prepared showing the live coding / demo

**Comment by Ankcorn** [comment]
👏 👏

This is awesome! I'm bookmarking this for future use thanks rob. This is going to be really helpful it should totally be shared where it will get more 👀 than this issue. Thank you

**Comment by robtweed** [comment]
You're welcome!  I hope it helps :-)

**Comment by robtweed** [comment]
One final thought: like any rules, feel free to break them, but I'd suggest only do so if you have the experience and/or confidence to do so.  eg if you look at the video of my talk and count the number of slides I used you'll be shocked - there were over 100!  But if you watch, I'm constantly skipping through them to provide a backdrop to what I'm saying.  Many of the slides are actually builds on a previous one, but even so!  A lot of rehearsal and thought went into each one and I stuck exactly to how I'd rehearsed it on the day.  If they're relevant and have a "so what?", a rapid succession of a lot of slides can be a good way of retaining the audience's attention and adding a feeling of pace to the presentation.  But only do it if you *know* you can get through them all within the allotted time!  Nothing worse than being cut off by the chairperson and having a load of slides you wanted to show, particularly if you didn't reach the killer "so what?" message slide in your deck!  And also remember that lots of slides can be too overwhelming - there's only so much your audience can absorb in 20 minutes!  Until you have the experience and confidence, keep things simple and tightly focused.  Less is almost always more when it comes to presentations!

**Comment by robtweed** [comment]
And, like a good meal, it's usually better to leave the audience wishing for more rather than going away bloated!

**Comment by Ankcorn** [comment]
Thanks Rob. I'm going to summarise this at our workshop. It's totally awesome. Thanks for all the time you put into writing it.

You should totally immortalise it as a blog post or feel free to submit a PR to add it to the speakers GitHub repo! I for one will be referring to this frequently before any future talks. A GitHub issue is far too temporary for something this legendary!

**Comment by GiacomoSorbi** [comment]
I would also add that having a short break might help: unless there is an active networking session for job seekers (which is not always there), 25 mins of break are far too much, IMHO: most people lose focus/mindset to follow further talks.

10 mins are more than enough to grab some more food or drinks or catch up with other attendants, and 15 mins of extra sleep (going home earlier) are always welcome.

**Comment by Ankcorn** [comment]
@lnug/organisers thoughts? I think we could do a 10-15 minute break. It would be fairer for people who need to get home. 

It also gives me more pub time! 🍻

**Comment by hewIngram** [comment]
Yeah I can see this - I think if we're stricter we can keep the break tighter around the jobs table session (or whatever else we run) that might work - theres ample time to chat before/after so it could work.

Lets try it next month and see?
